components/checkbox/index.css

```diff
@@ -352,7 +369,7 @@ governing permissions and limitations under the License.
     }
 
     /* Focus State */
-    &:focus-ring {
+    &:focus-visible {
```

Comment:
I'm wondering if we need this rule at all (`.spectrum-Checkbox--emphasized:focus-visible`). Is there a case where that could even get triggered? All the other focus rules are on the input. Clicking the parent label that wraps the input focuses the input.